firmware/ubirch/m66/m66_http.h

```diff
@@ -53,6 +53,37 @@ int modem_http_prepare(const char *url, uint32_t timeout);
  */
 int modem_http(http_method_t op, size_t *res_size, uint32_t timeout);
 
+
```

Comment:
i still have to document it, this was jus for reference for future documentation